.github/dependabot.yaml

```diff
@@ -0,0 +1,42 @@
+version: 2
```

Comment:
Need to run make update && make verify to get the  license agreement added otherwise /lgtm